### PR TITLE
PipTest: Update the expected outputs

### PR DESCRIPTION
**Pull request by fviernau**
Signed-off-by: Frank Viernau <frank.viernau@here.com>

### CODE REVIEW

File: analyzer/src/funTest/assets/projects/synthetic/pip-expected-output.yml [hunk 1]
```diff
@@ -14,21 +14,15 @@ project:
     path: ""
   vcs_processed:
     type: "Git"
-    url: "<REPLACE_URL>"
-    revision: "<REPLACE_REVISION>"
-    path: "<REPLACE_PATH>"
+    url: "ssh://git@github.com/heremaps/oss-review-toolkit.git"
```
Comment (sschuberth) [inline analyzer/src/funTest/assets/projects/synthetic/pip-expected-output.yml]: These should be templates.

File: analyzer/src/funTest/assets/projects/synthetic/pip-expected-output.yml [hunk 2]
```diff
@@ -14,21 +14,15 @@ project:
...
     dependencies:
     - id: "PyPI::Flask:1.0"
-      dependencies:
-      - id: "PyPI::Jinja2:2.10.1"
-        dependencies:
-        - id: "PyPI::MarkupSafe:1.0"
-      - id: "PyPI::Werkzeug:0.14.1"
-      - id: "PyPI::click:6.7"
-      - id: "PyPI::itsdangerous:0.24"
+    - id: "PyPI::pkg-resources:0.0.0"
```
Comment (sschuberth) [inline analyzer/src/funTest/assets/projects/synthetic/pip-expected-output.yml]: I thought this should not be listed according to https://github.com/heremaps/oss-review-toolkit/pull/1702...

**Comment by mnonnenmacher** [comment]
Commit message nit: Should be "expected results"

**Comment by fviernau** [comment]
somehow my machine produced different results, not sure what the mistake was but this PR is not needed at all.

**Pull request closed without merging**